mqtt/mqttd/Cargo.toml

```diff
@@ -8,7 +8,9 @@ edition = "2018"
 atty = "0.2"
 failure = "0.1"
 futures-util = { version = "0.3", features = ["sink"] }
-tokio = { version = "0.2", features = ["dns", "macros", "rt-threaded", "signal", "stream", "tcp", "time"] }
+tokio = { version = "0.2", features = ["dns", "macros", "rt-threaded", "signal", "stream", "tcp", "time", "fs"] }
```

Comment:
you can safely remove "fs" from features if we are not going to use async filesystem operations